qa/suites/orch/cephadm/upgrade/3-upgrade/simple.yaml

```diff
@@ -1,3 +1,9 @@
+overrides:
+  ceph:
+    log-ignorelist:
+      - MON_DOWN
```

Comment:
hitting a different variation of this same thing that just doesn't actually have MON_DOWN in the line
```
"2024-02-27T06:47:33.821991+0000 mon.a (mon.0) 200 : cluster [WRN] Health detail: HEALTH_WARN 1/3 mons down, quorum a,c" in cluster log
```

Reply:
seems to be yet another way this can present itself
```
cluster [WRN] mon.b (rank 2) addr [v2:172.21.15.104:3300/0,v1:172.21.15.104:6789/0] is down (out of quorum)" in cluster log
```